who_is_active.sql

```diff
@@ -1383,11 +1383,28 @@ BEGIN;
 
 		--Used for the delta pull
 		REDO:;
-		
-		IF
-			@get_locks = 1
-			AND @recursion = 1
-			AND @output_column_list LIKE '%|[locks|]%' ESCAPE '|'
+
+		DECLARE
+			@process_locks BIT =

```

Comment:
I put together both of these so that they can be checked below and we can only populate the @skip_databases table if necessary.